src/components/LeafletMap.js

```diff
@@ -319,6 +357,7 @@ class LeafletMap extends Component {
           this.removeHighlight();
```

Comment:
I don't believe you should be executing the same method that produces the same effect multiple times when doing it once should suffice

Reply:
I'm not sure what the will affect, this is used just once

Reply:
You are correct. Retracting...